olm-catalog/serverless-operator/project.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
     net_kourier_artifacts_branch: release-v1.14
     net_istio: knative-v1.14
     net_istio_artifacts_branch: release-v1.14
-    redhat-knative-istio-authz-chart: 1.34.0
+    redhat-knative-istio-authz-chart: 1.35.0
```

Comment:
does this already exist?

Reply:
Nope https://github.com/openshift-knative/knative-istio-authz-chart/branches/all. I guess we install from github not a release branch anyway https://github.com/openshift-knative/serverless-operator/blob/main/hack/generate/mesh-auth-policies.sh#L27

Reply:
The chart branch is created after we cut S-O repo, then S-O main will use charts main as next version in the interim.   At least that's my understanding of the setup.

Reply:
yes, it exists, it's on main https://github.com/openshift-knative/knative-istio-authz-chart/blob/main/Chart.yaml